handsontable/src/plugins/hiddenColumns/hiddenColumns.js

```diff
@@ -365,6 +382,19 @@ export class HiddenColumns extends BasePlugin {
     });
   }
 
+  /**
+   * Remove the indicator elements from the provided column header element.
+   *
+   * @param {HTMLElement} TH Column header element.
+   */
+  #clearIndicatorElements(TH) {
+    Array.from(TH.querySelectorAll(
+      `.${AFTER_INDICATOR_CLASSNAME}, .${BEFORE_INDICATOR_CLASSNAME}`
+    )).forEach((element) => {
+      removeChildIfExists(TH, element);
```

Comment:
```suggestion
      element.remove();
```
I wonder wouldn't this be enough instead of using a helper?